### DISCUSSION TITLE
Manually have to select every stock and apply indicators and set their values just to find that if there's a crossover or not.

### DISCUSSION THREAD

**Discussion by Vaibhavv14** [ShubhamNagure/BhavCopy-Report-Analysis#14] [Ideas]
Problem statement
 Manually have to select every stock and apply indicators and set their values just to find that if there's a crossover or not.

Indicators used : 50 Exponential Moving average and 200 EMA 

Basic strategy : 
>look for the opportunities where 50 EMA has crossed 200 EMA "above" to make buying opportunity(Golden crossover)
>If 50 EMA crosses 200 EMA "Below" then we need to look for selling opportunity.(Death crossover)

*50  EMA is average value of last 50 candles.
*200 EMA is avg of last 200 candles.

Solution:

> Creating a algorithm that runs a scan from predefined list of stocks to check for the "crossover" and sends an alert.

**Comment by ShubhamNagure** [comment]
Thanks, Vaibhav for bringing this up.

It's a great idea to keep track of all stock at EOD to apply the quote indicator logic to them which will return us a list of stock (only nifty 50).

Being said that, in order to calculate the EMA for nifty stocks we will be leveraging below formula.

EMA = [ CMP X e ] + [ previous EMA X (1 - e) ]

where,
CMP = current market price
e = exponential quotient

in order to calculate the 'e' formula is.

e= 2 / (n+1)

where n = number of days.


> I will take this to issue dashboard and will implement the said logic. Please feel free to comment.
